Description: desktop file updates
 removes deprecated encoding key, and updates category
Author: Jasmine Hassan <jasmine.aura@gmail.com>
Last-Update: 2012-10-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ccsm.desktop.in
+++ b/ccsm.desktop.in
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=CompizConfig Settings Manager
 _Comment=Configure Compiz with CompizConfig
 Icon=ccsm
 Exec=ccsm
 Terminal=false
 Type=Application
-Categories=Compiz;Settings;DesktopSettings;
+Categories=Settings;DesktopSettings;
 StartupNotify=true
